src/js/index.js

```diff
@@ -1,7 +1,74 @@
+const baseUrl = "https://js-todo-list-9ca3a.df.r.appspot.com";
+
+
 const onUserCreateHandler = () => {
-  const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+    const userName = prompt("추가하고 싶은 이름을 입력해주세요.");
+    if (userName == null || userName == undefined)
+        return;
+    if (userName.length < 2)
+        alert("사용자 이름은 최소 2글자 이상입니다.");
+
+    // 사용자 추가하기
+    $.ajax({
+        type: "POST",
+        url: `${baseUrl}/api/users`,
+        data: { "name": userName },
+        success: function(result) {
+            getAllUser(result._id);
+        }
+    });
 }
 
-const userCreateButton = document.querySelector('.user-create-button')
-userCreateButton.addEventListener('click', onUserCreateHandler)
+//USERlIST 호출
+function getAllUser(id) {
+    $.ajax({
+        type: "GET",
+        url: `${baseUrl}/api/users`,
+        success: function(result) {
+            createUserBtn(result, id);
+        }
+    })
+}
+
+//버튼생성
+function createUserBtn(userListArr, focusId) {
+    let userListDiv = document.getElementById("user-list");
+    document.querySelector('.active').className = 'ripple';
+
+    for (var i = 0; i < userListArr.length; i++) {
+        let button = document.createElement("button");
+
+        if (userListArr[i]._id === focusId) {
+            button.setAttribute('class', 'ripple active');
+        } else {
+            button.setAttribute('class', 'ripple');
+        }
```

Comment:
```suggestion
        button.setAttribute('class', `ripple ${ userListArr[i]._id === focusId && 'active' }`);
```
리터럴 표기법을 사용해 이렇게도 가능하답니다~ 😊